tests/unit/models/github-member-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
```

Comment:
i left this and the serializer test in to be consistent but since none of these tests actually do anything I would be 👍 on removing them all at some point

Reply:
Yea it's fine to leave this for a project-wide cleanup.